packages/tocco-ui/src/Upload/UploadProgress.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Preview from '../Preview'
+import {bytesToSize} from './helpers'
+
+const UploadProgress = props => (
+  <div>
+    <Preview
+      className="transparent"
+      style={{maxWidth: '200px'}}
```

Comment:
Should we use a fixed width?

Reply:
for the moment i would say yes. stylers (pale or nik) should add proper styling later.